string-manipulation-2/src/swap-chars.js

```diff
@@ -0,0 +1,14 @@
+/* exported swapChars */
+function swapChars(firstIndex, secondIndex, string) {
+  var newString = '';
+  for (var i = 0; i < string.length; i++) {
+    if (i !== firstIndex && i !== secondIndex) {
+      newString += string[i];
+    } if (i === firstIndex) {
```

Comment:
when using multiple if statements I would alter the formatting to keep them on the same position:
```javascript
if(){
}
if(){
}
```
the current formatting looks more like `if else` statements